uiplib/scrape.py

```diff
@@ -36,23 +36,27 @@ def get_images(url):
     '''
     scrape images from /r/wallpapers
     '''
-    soup = make_soup(url)
-    # this makes a list of bs4 element tags
-    thumbnails = soup.find_all("a", class_="thumbnail", href=True)
+    image_links = []
+    p_url = url
+    while len(image_links) < NUMBER_OF_IMAGES_TO_PARSE:
```

Comment:
Suppose I want to download some 200 wallpapers, if I run this check, I don't think this loop is ever gonna end, because the DOM might not contain those many images